vignettes/release_strategy.Rmd

```diff
@@ -41,37 +41,29 @@ updates via the community meetings, especially any anticipated breaking changes.
 
 ## Quarterly Release
 
-A package release is done in five parts:
+A package release is done in four parts:
 
-1) Create a Pull Request from `devel` into the `main` branch.  Issues identified in this Pull Request should have work done in separate branches and merged once again into `devel`.
-1) Verify that all CI/CD checks are passing for the `devel` into the `main` Pull Request, merge and then bundle up and send off to CRAN. See the [chapter](https://r-pkgs.org/release.html#decide-the-release-type) in R Packages for more details.
+1) Verify that all CI/CD checks and then bundle up and send off to CRAN. See the [chapter](https://r-pkgs.org/release.html#decide-the-release-type) in R Packages for more details.
```

Comment:
Don't we need a PR into `main` were we update the version number (remove `.9000` part and increase version)?

Reply:
that depends on the workflow one uses. the PR checklist from usethis, has you keep your version bump to the release version local until CRAN accepts. But others do different things.